build/metrics/binary_size.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+import subprocess
+import urllib.parse
+
+# ELF analysis
+
+def loadable_sections(elf_file, address_prefix = ""):
+  objdump_section_headers_pattern = re.compile("^\s+\d+\s+(\.[\w\.]+)\s+([0-9a-f]+)\s+([0-9a-f]+)\s+("+address_prefix+"[0-9a-f]+)\s+([0-9a-f]+).*LOAD", flags=re.MULTILINE)
```

Comment:
Maybe add a comment to explicit this to the reader?

Reply:
Well… this is an unfair request :-D This exact same code already made it to the tree without any comment whatsoever. See `build/device/elf2dfu.py` 😄